libs/libvtrutil/src/vtr_log.cpp

```diff
@@ -14,3 +18,33 @@ void set_log_file(const char* filename) {
 }
 
 } // namespace vtr
+
+void add_warnings_to_suppress(std::string function_name) {
+    warnings_to_suppress.insert(function_name);
+}
+
+void set_noisy_warn_log_file(const char* log_file_name) {
```

Comment:
Should comment what you're doing in this function (making sure the log file is empty).

Reply:
Done. I've added the comment above the function declaration